.github/workflows/ci.yml

```diff
@@ -300,6 +291,31 @@ jobs:
         run: |
           /bin/sh -c "ALL_TESTS=1 ASAN_OPTIONS=${ASAN_OPTIONS} TSAN_OPTIONS=${TSAN_OPTIONS} VALGRIND=${VALGRIND} RETRIES=${RETRIES} make check -C test || (test/filter-suite-log.sh test/test-suite.log; exit 1)"
 
+  clang-tidy:
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:40
+
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v4
+
+      - name: Install packages
+        run: |
+          .github/workflows/linux-ci-helper.sh fedora:40
```

Comment:
This line should be `.github/workflows/linux-ci-helper.sh ${{ matrix.container }}`

Reply:
I tried matrix.container and container.image but neither worked.

Reply:
Ok, something's weird, but I'll merge this and check it later.

Reply:
@gaul I'm sorry about my comment, so blow is additional information.
Since this JOB does not use a matrix, I confirmed that the container name cannot be obtained as a GHA variable.
(I was able to find `FBR=f40`, `DISTTAG=f40container`, `FGC=f40` in the environment variables, but these should not be used.)
So there was no need to modify it as it was.